lib/common.go

```diff
@@ -25,6 +25,22 @@ import (
 
 const defaultWorkPath = ".acbuild"
 
+// ErrNotFound is returned when acbuild is asked to remove an element from
+// a list and the element is not present in the list
+var ErrNotFound = fmt.Errorf("element to be removed does not exist in this ACI")
+
+// GetErrorCode will return the recommended exit code for acbuild given an error
+func GetErrorCode(err error) int {
```

Comment:
Interesting approach. I think this function should exist in the CLI package instead though (but still key off the lib.Err types)